test/com/facebook/buck/cli/testdata/buck_run/pwdbase.py

```diff
@@ -2,6 +2,6 @@
 import sys
 
 cwdbase = os.path.basename(os.getcwd())
-print cwdbase
+print (cwdbase)
 sys.exit(0 if cwdbase == 'subdir' else 1)
 
```

Comment:
Python3 has changed its import precedence and using same name as a python module gives you an error "ImportError: No code object available for ...". Since `pwd` is a python module, we can no longer use the file name `pwd.py`.